building/devices/index.rst

```diff
@@ -6,7 +6,8 @@ Device specific information
 
 .. toctree::
     :maxdepth: 1
-
+	       
```

Comment:
When looking at the `git diff` I see a lot of blank spaces on "empty" lines as well as after the last character on quite a few lines in the patch set. Please remove those (this line is one of them).

Reply:
um sorry, my emacs settings messed up